USAGE.md

```diff
@@ -275,9 +275,11 @@ $ aws-vault login work
 
 ### Removing stored sessions
 
-If you want to remove sessions managed by `aws-vault` before they expire, you can do this with the `--sessions-only` flag.
+If you want to remove sessions managed by `aws-vault` before they expire, you can do this with `aws-vault clear-cache` command.
+
+You can also specify a profile to remove sessions for this profile only.
 ```shell
-aws-vault remove <profile> --sessions-only
+aws-vault clear-cache [<profile>]
```

Comment:
How about square to make it clear that it's an optional argument? See https://en.wikipedia.org/wiki/Command-line_interface#Command_description_syntax for the closest thing I can find to a standard.

```suggestion
aws-vault clear-cache [profile]
```